.changeset/slow-cats-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/sitemap": patch
+---
+
+Rever https://github.com/withastro/astro/pull/9846
```

Comment:
A description here to explain why it was reverted would be great in case any users are using the feature!